recipes/flatbuffers/all/conanfile.py

```diff
@@ -2,7 +2,7 @@
 from conan.tools.apple import is_apple_os
 from conan.tools.build import check_min_cppstd, valid_min_cppstd
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
-from conan.tools.files import apply_conandata_patches, collect_libs, copy, get, replace_in_file, rmdir
+from conan.tools.files import export_conandata_patches, apply_conandata_patches, collect_libs, copy, get, replace_in_file, rmdir
```

Comment:
required_conan_version to 1.52.0 please

you can also take advantage of this PR to use rm_safe to remove fPIC in configure (conan 1.53.0)

Reply:
Thank a lot!
Fixed it.